src/conversions/sample_rate.rs

```diff
@@ -192,7 +191,7 @@ where
             debug_assert!(self.next_frame.is_empty());
 
             // draining `self.current_frame`
-            if self.current_frame.len() >= 1 {
+            if !self.current_frame.is_empty() {
```

Comment:
I don't like this change, can you keep it?

Reply:
There are quite a few other .is_empty() such as literally at the top of this inline diff at 191. Do I replace them with .len() >= 1 as well? Also what about .len() > 0 maybe?

Reply:
Alright let's go with your code then.